Add Custom VCC Blueprint (#31)
madewithkoji/docs: src/asciidoc/docs/blueprints/blueprints.adoc

* <<magazine-cover-blueprint#, Magazine cover>> - Put your own picture and headline on a magazine by implementing customizable images and text.
* <<vote-counter-blueprint#, Vote counter>> – Create a multi-user, interactive vote counter by implementing a Koji database and real-time dispatch.

== Build your custom VCC

> * <<cat-selector-blueprint#, Cat Selector VCC>> – Create a custom Koji VCC which lets remixers choose from a list of options retrieved from a third-party API.

Reviewer: ```suggestion
* <<cat-selector-blueprint#, Cat Selector VCC>> – Create a custom Koji VCC that lets remixers choose from a list of cat breeds retrieved from a third-party API.
```
